### ISSUE TITLE
More graceful abort when rate limit is hit on list import

### ISSUE THREAD

**Issue by pruvisto** [pruvisto/debirdify#14]

**Comment by cooljeanius** [comment]
yeah, like just skip that list, and then either move onto the next one, or wait a bit and try again, or just return all the results that have already been found before the limit was hit
